Update Cohort-call-facilitator-Guide.md to include latest addition (#25)
open-life-science/policies-procedures-and-docs: cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md

# OLS Cohort Facilitators

We are very excited to create this role in OLS for the returning members from the previous cohorts of OLS.

We want to invite support from our community members in making the cohorts successful.

At the same time, our hope is to share our practices for training and planning so you can take them into your projects and community.

Our cohort call facilitators share some of the cohort call-related management roles with the OLS team. 

Please identify which week you have signed up for and who is the cohort call lead.

Our cohort calls vary in time from week to week: Please see schedules from our website here: [[https://openlifesci.org/](https://openlifesci.org/)]([https://openlifesci.org/](https://openlifesci.org/)).

> For any questions or concerns, please contact Community Coordinator, Paz Berlando via Slack.

Reviewer: ```suggestion
For any questions or concerns, please contact Community Coordinator, Paz Bernaldo via Slack.
```

You can also reach out to other OLS team members via Slack or email at any time.

## Cohort Facilitator Roles

We have two facilitator roles per call. In some cases, it’s possible that the same person has signed up for both roles. 

1. **Call Facilitator** reviews the cohort call notes and attends the cohort call to help the cohort call lead in making sure that a cohort call in a smooth, kind, and timely manner
2. **Recording Facilitator** reviews the recording from the cohort call by checking the otter.ai transcription for accuracy and sharing the video via Youtube

**Please ask for the signup sheet where you can indicate your availability for 1 or multiple of these roles**

> As of August 2023, both facilitator roles are compensated with $ 50 USD per hour.

Reviewer: ```suggestion
As of August 2023, both facilitator roles are compensated with $ 50 USD per hour, with a maximum of 1.5 hours per cohort call or 3 hours for transcription.
```

idk if we have this but maybe we should?

1. Ask the call lead for co-host rights and check if the call will be recorded on the cloud (if applicable, because some calls are not recorded, or parts of them are not recorded, like in week 13 for the skills-up call on self-help). 
2. Test if you can create breakout rooms (for discussion parts).
3. Help manage participants by letting them in (if the waiting room is enabled) and mute them if their mic is on when they are not speaking to avoid background noise.
4. Keep an eye on the chat and respond to questions. You can also privately coordinate with the participants or cohort call lead as needed, for example, if someone needs help with addressing any confusion or interruptions for attendees.
5. Remind newcomers via the chat to add an_ S _or a _W_ in front of their Zoom name (S for spoken; W for written), for the breakout rooms. Privately coordinate with them if they have not indicated ‘W’ for written and ‘S’ for speaking to allow the creation of breakout rooms.
6. Share the Etherpad link in the chat for the participants who arrive late and remind them to add their names in the roll call section of the Etherpad.

> 7. Please also help take some notes per talk when you are not chairing.

Reviewer: ```suggestion
7. Please also help take some notes about the talks when you are not chairing.
```

8. During the call, share the Etherpad’s line number where the participants should respond to a written reflection, questions or comments for speakers, prompts for the breakout discussion or notes section from breakout sessions.
9. Create the breakout rooms if needed, considering people’s choice for spoken or written rooms. 

> 10. Chair the session that you have agreed to (you would have coordinated this in advance with the call leads or community coordinator)

Reviewer: ```suggestion
10. Chair the session that you have agreed to (you would have coordinated this in advance with the call leads or community coordinator)

#### Chairing notes: 
- If you are chairing a speaker session, make sure to pay attention to time, and ask the speaker to wrap up if they look like they're going to over-run their talk time. 
- Always think of questions for the speakers. Often no-one wants to be the first person to ask, so a good call host can "break the ice" by asking one themselves. 
```

### Recording Facilitator

**These facilitators hold the post-call responsibility mainly.**

> What you would need: 

Reviewer: ```suggestion
What you will need: 
```
